README.md

```diff
@@ -378,6 +378,53 @@ $ cargo test
 
 from the repository root.
 
+### Tips
+
+#### Windows Powershell
+
+##### Powershell Profile
+
+To customize powershell on start-up there is a special powershell script that has to be created.
+In order to find its location run command `Get-Command $profile | Select-Object -ExpandProperty Definition`
+
+But its typical location should be following: `C:\Users\<your_user_name>\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1`
+If there is no directory `WindowsPowerShell` then it must be created too.
+
```

Comment:
The location of the profile can be gotten just with `$profile`; no need for that complex command. And you can check to see if a profile exists with `test-path $profile`. You might want to link to the [official docs on profiles](https://technet.microsoft.com/en-us/library/bb613488(v=vs.85).aspx), which cover the various profile locations and have instructions on how to create one if it doesn't exist (`new-item -path $profile -itemtype file -force`).